packages/web-app-files/src/services/folder/loaderFavorites.ts

```diff
@@ -4,9 +4,15 @@ import { useTask } from 'vue-concurrency'
 import { DavProperties } from 'web-pkg/src/constants'
 import { buildResource } from '../../helpers/resources'
 import { isLocationCommonActive } from '../../router'
+import { Store } from 'vuex'
+import get from 'lodash-es/get'
 
 export class FolderLoaderFavorites implements FolderLoader {
-  public isEnabled(router: Router): boolean {
+  public isEnabled(store: Store<any>): boolean {
+    return get(store, 'getters.capabilities.files.favorites', true)
```

Comment:
That's highly individual. Some capabilities exist in oc10, some don't. In general we can define new capabilities in oCIS easier than in oc10 (because of release cycles). So most of the time we have to put the expected oc10 value as default. Feels like too many comments to have that on every capability usage. :-/